### DISCUSSION TITLE
Are there tagged versions other than :latest ?

### DISCUSSION THREAD

**Discussion by brad-4k** [gis-ops/docker-valhalla#66] [General]
Hi I'm,

I'm looking for a tagged version that would represent :latest or just before :latest.

Is there such a thing?

Thanks!

**Comment by ImreSamu** [comment]
Workaround ..

You can use the older image `Digest` ( sha256 )
- step1 : find a "green checkbox"[ in the commits list.](https://github.com/gis-ops/docker-valhalla/commits/master)
- click the "green checkbox" - select **"publish latest / publish (push) "** - "details" 
![image](https://user-images.githubusercontent.com/2217320/165803155-602ac419-36b2-4e8c-8df6-73a1f9d7158d.png)

- [open](https://github.com/gis-ops/docker-valhalla/runs/5989511000?check_suite_focus=true#step%3A5%3A354=) the "Build container image" part - and on the bottom - find the "Digest"

![image](https://user-images.githubusercontent.com/2217320/165802446-15f6258e-0d74-4342-8fbb-4683b8504886.png)

and now you can use the digest ..
`docker pull gisops/valhalla:latest@sha256:03d1756265e5a66cde250ee9d4e90395ad22447d905a525e3b911eac3427313b`


EDIT:
You can find the current latest digest in the docker hub
https://hub.docker.com/layers/valhalla/gisops/valhalla/latest/images/sha256-5d28ecf3dc6fc57f900096dd996d2ca37183ae64bf06d9bb5c5e757fa72acf85?context=explore

![image](https://user-images.githubusercontent.com/2217320/165804241-08af7153-50a2-43d2-927e-411576b58dd8.png)

**Reply by nilsnolde** [reply]
great thanks @ImreSamu !
